frontend/src/app/views/screenskit/reports/reportsIndividual/FormIndividual.jsx

```diff
@@ -0,0 +1,223 @@
+import {
+    Box,
+    Button,
+    Divider,
+    FormControl,
+    InputLabel,
+    MenuItem,
+    Select,
+    Tooltip,
+    Typography,
+} from "@mui/material";
+import styled from "@emotion/styled";
+import { useEffect, useMemo, useState } from "react";
+import instance from "../../../../../axios"
+import { FcPrint } from "react-icons/fc";
+import TableIndividual from "./TableIndividual";
+import { Link } from "react-router-dom";
+const categorys = [
+    {
+        "id": 1,
+        "name": "Categoria Especial - Modalidade Arremesso",
+    },
+    {
+        "id": 2,
+        "name": "Categoria Comum - Modalidade Arremesso e Corrico",
+
+    }
+];
+
+const URL_REPORTS_CATEGORY1 = "http://localhost:8000/api/reports/categories/1/individual"
+const URL_REPORTS_CATEGORY2 = "http://localhost:8000/api/reports/categories/2/individual"
```

Comment:
```suggestion
const URL_REPORTS_CATEGORY1 = "http://localhost:8000/api/reports/categories/1/individual"
const URL_REPORTS_CATEGORY2 = "http://localhost:8000/api/reports/categories/2/individual"
```

Mano, o ideal aqui seria usar aquela env REACT_APP_API_BASE_URL

Reply:
Ja adicionei a url na variavel de ambiente no arquivo .env e no .env.example tbm